test/data/BM_example.yaml

```diff
@@ -0,0 +1,86 @@
+description: |-
+  This is the minimal example tweaked from Hydrogen-Oxygen 
+  submechanism extracted from GRI-Mech 3.0. It should only 
+  be used for testing the Blowers Masel reaction class in 
+  gas phase.
+  
+generator: ck2yaml
+input-files: [h2o2.inp, gri30_tran.dat]
+cantera-version: 2.5.0
+date: Wed, 11 Dec 2019 16:59:04 -0500
+
+units: {length: cm, time: s, quantity: mol, activation-energy: cal/mol}
+
+phases:
+- name: ohmech
+  thermo: ideal-gas
+  elements: [O, H, Ar, N]
+  species: [H2, H, O, OH, H2O, O2]
```

Comment:
You can eliminate the need to duplicate species definitions from `h2o2.yaml` by writing this as:
```yaml
  species: [{h2o2.yaml: [H2, H, O, OH, H2O, O2]}]
```
You'll just have to make a small modification to `test_negative_A_blowersmasel` to read the species definitions from the original file as well.